src/mlpack/core/tree/binary_space_tree/typedef.hpp

```diff
@@ -135,6 +135,20 @@ using MeanSplitBallTree = BinarySpaceTree<MetricType,
                                           bound::BallBound,
                                           MeanSplit>;
 
+template<typename MetricType, typename StatisticType, typename MatType>
+using RPTreeMax = BinarySpaceTree<MetricType,
+                                  StatisticType,
+                                  MatType,
+                                  bound::HRectBound,
+                                  RPTreeMaxSplit>;
+
+template<typename MetricType, typename StatisticType, typename MatType>
+using RPTreeMean = BinarySpaceTree<MetricType,
+                                  StatisticType,
+                                  MatType,
+                                  bound::HRectBound,
+                                  RPTreeMeanSplit>;
```

Comment:
Maybe you are right. These names are used in Dasgupta's paper "Random projection trees and low dimensional manifolds". I guess the mean split is the "default" since I saw that in another paper.